### ISSUE TITLE
Missing accessibility labels

### ISSUE THREAD

**Issue by gsilver** [RossGeerlings/PlasmaPup#1]
Text input for in the "Specify Additional..." modal is unlabeled
List underneath the "Specify Additional..." button is unlabeled
